hermes.podspec

```diff
@@ -0,0 +1,45 @@
+module HermesHelper
+  # BUILD_TYPE = :debug
+  BUILD_TYPE = :release
+
+  def self.command_exists?(bin)
+    "command -v #{bin} > /dev/null 2>&1"
+  end
+
+  def self.configure_command
+    "./utils/build/configure.py #{BUILD_TYPE == :release ? "--distribute" : "--build-type=Debug"} --cmake-flags='-DHERMES_ENABLE_DEBUGGER:BOOLEAN=true -DHERMES_ENABLE_FUZZING:BOOLEAN=false -DHERMES_ENABLE_TEST_SUITE:BOOLEAN=false -DCMAKE_INSTALL_PREFIX:PATH=../destroot' build"
+  end
+end
+
+Pod::Spec.new do |spec|
+  spec.name        = "hermes"
+  spec.version     = "0.5.0"
```

Comment:
Can you mention that this is a fourth place to update the version in the top level CMakeLists.txt:

https://github.com/facebook/hermes/blob/670817d9b27ded6ed73c086abffcb7b6220f41d3/CMakeLists.txt#L55-L59